packages/zowe-explorer-api/src/profiles/ZoweExplorerZosmfApi.ts

```diff
@@ -452,15 +452,15 @@ export namespace ZoweExplorerZosmf {
      * An implementation of the Zowe Explorer Command API interface for zOSMF.
      */
     export class CommandApi extends CommonApi implements MainframeInteraction.ICommand {
-        public issueTsoCommandWithParms(command: string, parms: zostso.IStartTsoParms): Promise<zostso.IIssueResponse> {
+        public issueTsoCommandWithParms(command: string, parms: zostso.IStartTsoParms, useNewTsoApis?: boolean): Promise<zostso.IIssueResponse> {
+            if (useNewTsoApis) {
+                return zostso.IssueTso.issueTsoCmd(this.getSession(), command, { addressSpaceOptions: parms });
```

Comment:
We might want to add a test case to verify that `issueTsoCommandWithParms` calls the new API when `useNewTsoApis` is `true`.

Reply:
Resolved in ee113dff0